quarkchain/cluster/test_gRPC_client.py

```diff
@@ -0,0 +1,45 @@
+import grpc
+import grpc_testing
+import unittest
+
+import gRPC_pb2
+from gRPC_server import ClusterSlave
+
+
+class TestGreeter(unittest.TestCase):
```

Comment:
test case naming wrong. also, I didn't see any test involving the client you implemented